<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtestceplibMainWindow</class>
 <widget class="QMainWindow" name="QtestceplibMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test ceplib</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="m_ActionReadSettings"/>
    <addaction name="m_ActionWriteSettings"/>
    <addaction name="m_ActionLoadTIFFImage"/>
    <addaction name="m_ActionTestHDF"/>
    <addaction name="m_ActionTestNexus"/>
    <addaction name="m_ActionTestHDFSlab"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="m_ActionReadSettings">
   <property name="text">
    <string>Read Settings...</string>
   </property>
   <property name="toolTip">
    <string>Read Settings</string>
   </property>
  </action>
  <action name="m_ActionWriteSettings">
   <property name="text">
    <string>Write Settings...</string>
   </property>
   <property name="toolTip">
    <string>Write Settings</string>
   </property>
  </action>
  <action name="m_ActionLoadTIFFImage">
   <property name="text">
    <string>Load TIFF Image...</string>
   </property>
  </action>
  <action name="m_ActionTestHDF">
   <property name="text">
    <string>Test HDF Library</string>
   </property>
  </action>
  <action name="m_ActionTestNexus">
   <property name="text">
    <string>Test Nexus Library</string>
   </property>
  </action>
  <action name="m_ActionTestHDFSlab">
   <property name="text">
    <string>Test HDF Slab Output</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
